.travis.yml

```diff
@@ -22,6 +22,9 @@ script:
 - make crossbuild
 after_script:
 - goveralls -coverprofile=.profile.cov
+- docker pull astj/mackerel-rpm-builder:c5
+- docker pull astj/mackerel-rpm-builder:c7
+- make rpm rpm-kcps rpm-stage
```

Comment:
I'll revert these changes in `.travis.yml` later. (I changed them just to check `make` works fine in travis.)